ios/NodeRunner.swift

```diff
@@ -53,6 +52,31 @@ class NodeRunner {
         self.keyStore = ks
     }
     
+    func getKeyStore() -> GethKeyStore? {
+        return self.keyStore
+    }
+    
+    func findAccount(rawAddress: String) throws -> GethAccount? {
```

Comment:
why do we need this? we can't index by the address?

Reply:
We can but it would mean to create and maintain this index inside of this bridge module. My intention was to not hold any state inside of this code and just rely on what's exposed from `celo-blockchain` which would be the place where state lives.

I chose this way because there currently isn't any way to lookup by address, but the overhead of this op is pretty small. Alternatively we can extend things on the `celo-blockchain` side with more convenience methods and reduce the footprint here but it felt like it would be harder to wrap this up because in order to function it would need to sync 3 codebases `mobile` -> `react-native-geth` -> `celo-blockchain`, so throughout the task I erred towards not changing `celo-blockchain` if it's possible to get away with just using what's currently available & exported.